doc/man3/SSL_CTX_set_mode.pod

```diff
@@ -105,6 +105,22 @@ Enable asynchronous processing. TLS I/O operations may indicate a retry with
 SSL_ERROR_WANT_ASYNC with this mode set if an asynchronous capable engine is
 used to perform cryptographic operations. See L<SSL_get_error(3)>.
 
+=item SSL_MODE_NO_KTLS_TX
```

Comment:
Nit: Please add this to the HISTORY section below.

Reply:
Done